File: ios/USTAQuery/Models/Registration.swift
import Foundation

struct Registration: Codable, Identifiable {
    let id: Int
    let tournament: Tournament
    let matchType: String
    let divisionName: String
    let player1: PlayerSummary
    let player2: PlayerSummary?
    let seed: Int?
    let status: String
    let registrationDate: String?
}

File: android/app/src/main/java/com/usta/query/data/model/Registration.kt
package com.usta.query.data.model

data class Registration(
    val id: Int,
    val tournament: Tournament,
    val matchType: String,
    val divisionName: String,
    val player1: PlayerSummary,
    val player2: PlayerSummary? = null,
    val seed: Int? = null,
    val status: String,
    val registrationDate: String? = null
)
